### ISSUE TITLE
UX/accessibility Issue - Cannot reach the community switches with tab on Community

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1398]
**Describe the bug**
All affordances should be available through tabbing through the site, however, community switches cannot be reached from the keyboard.

**To Reproduce**
1. Go to 'https://www.topcoder-dev.com/settings/profile'
2. Click on 'Community'
3. Tab through the page

**Expected behavior**
The user should be able to set the communities from the keyboard.

**Screenshots**
http://www.screencast.com/t/2xrEqB4Ht

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas

**Comment by ThomasKranitsas** [comment]
I don't think this is an issue... It's not a radio button thus it kinda makes sense to me to not being able to change the options with tab button.

**Comment by PereViki** [comment]
WCAG 2.0 Guidelines about keyboard accessibility: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#keyboard-operation
